README.md

```
@@ -2579,6 +2579,23 @@ Set a user property for use in Analytics:
 ```javascript
 FirebasePlugin.setUserProperty("name", "value");
 ```
+### Enable debug_mode to use DebugView
+You can find detailed information [here](https://firebase.google.com/docs/analytics/debugview?hl=es-419#android)
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)